CIP-1694/README.md

```diff
@@ -735,7 +737,6 @@ DRep registration certificates include:
 
 * a DRep ID
 * a deposit
-* a stake credential (for the deposit return)
```

Comment:
how does the deposit return work then?

Reply:
It's immediate & via transaction balancing, like for staking credentials.

Reply:
right. something that needs explanation (or i missed the explanation somewhere)

Reply:
This is explained right below this one, in the DRep retirement cert.